portfolio/pom.xml

```diff
@@ -35,7 +35,7 @@
         <version>2.2.0</version>
         <configuration>
           <!-- TODO: set project ID. -->
-          <deploy.projectId>YOUR_PROJECT_ID_HERE</deploy.projectId>
+          <deploy.projectId>amatasanchez-sps-summer20</deploy.projectId>
```

Comment:
Same about this change file being replicated.

Reply:
Fixed!